torch/csrc/onnx/init.cpp

```diff
@@ -0,0 +1,26 @@
+#include "torch/csrc/onnx/init.h"
+#include "torch/csrc/onnx/onnx.pb.h"
+
+namespace torch { namespace onnx {
+void initONNXBindings(PyObject* module) {
+  auto m = py::handle(module).cast<py::module>();
+  auto onnx = m.def_submodule("onnx");
```

Comment:
Please add a leading underscore. We import everything from _C as is to the namespace and it will clash with the Python module.